benchmarks/renderingViewInTemplate.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
```

Comment:
an example of how this benchmark goes:

```bash
[11:03:02] Running suite Rendering child views in template ...
[11:03:08]    child views in template x 3,289 ops/sec ±4.00% (82 runs sampled)
[11:03:14]    child views by replacing a node via parent view x 3,500 ops/sec ±2.63% (87 runs sampled)
[11:03:19]    child views by appending them via parent view x 4,726 ops/sec ±2.39% (86 runs sampled)
[11:03:19] Fastest test is child views by appending them via parent view at 1.35x faster than child views by replacing a node via parent view
```

this new functionality is currently slightly slower than using andReplace